page-archivo.php

```diff
@@ -0,0 +1,106 @@
+<?php
+get_header();
+
+$archivo_post_types = array(
+  'post',
+  'exposiciones',
+  'obra',
+  'journal',
+  'publicaciones',
+);
+
+// Get yearl url param
+$year_param = empty($_GET['año']) ? null : $_GET['año'];
```

Comment:
all fun and games until someones shit browser doesnt work and then the client complains